resample/bootstrap.py

```diff
@@ -215,15 +215,15 @@ def _fit_parametric_family(dist: stats.rv_continuous, sample: np.ndarray) -> Tup
         return np.mean(sample, axis=0), np.cov(sample.T, ddof=1)
 
     if dist == stats.t:
```

Comment:
We can just remove the first `if`, then the `else` clause at the end handles this.

Reply:
Fixed